librz/asm/arch/arc/gnu/arcompact-dis.c

```diff
@@ -3911,7 +3911,7 @@ ARCompact_decodeInstr (bfd_vma           address,    /* Address of this instruct
         {
           /* Branch instruction with 3 operands, Translation is required
              only for the third operand. Print the first 2 operands */
-          strncpy(buf, operand, sizeof (buf) - 1);
+          strncpy(buf, operand, sizeof (buf));
```

Comment:
I think this should be followed by a `buf[sizeof(buf)-1] = '\0';`

Reply:
Ok b14aa201a4035a76d37a14cb9d1fa674f8658c16